js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -140,7 +140,7 @@ var KeyboardNavigationController = core.ViewController.inherit({
             eventsEngine.off($element, eventUtils.addNamespace(pointerEvents.up, "dxDataGridKeyboardNavigation"), clickAction);
             eventsEngine.on($element, eventUtils.addNamespace(pointerEvents.up, "dxDataGridKeyboardNavigation"), clickSelector, clickAction);
 
-            that._initKeyDownProcessor(that, $element, that._keyDownHandler);
+            that._initKeyDownHandler($element, e => that._keyDownHandler(e));
```

Comment:
We can not create a new function.

`that._initKeyDownHandler($element, e => that._keyDownHandler(e));`
->
`that._initKeyDownHandler($element, that._keyDownHandler);`